### DISCUSSION TITLE
Assistance Needed with Recovering DownloadHelper Keys

### DISCUSSION THREAD

**Discussion by lucasdsgarcia** [aclap-dev/video-downloadhelper#1084] [License & Purchase requests]
### What's your issue?

Good evening,

I am reaching out to DownloadHelper support because I am unable to recover my two keys. I have entered the email addresses, but neither key is arriving. Both keys were purchased less than a year ago. Could you please assist me? I have attempted this more than five times at different intervals, but none of the keys have been received. I am certain these are the emails used for the purchase. Thank you in advance, and I look forward to your assistance.

Best regards,

Lucas Garcia

### The 10 first digits (only 10) of your license number or email address

lucasgarciadasilva@outlook.com.br

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

At the top of every page of this forum you'll see a link to the Wiki.  Click that.  That takes you to the closest thing we have to a user manual, something everybody should get in the habit of referring to regularly.  Its table of contents is a column of links down the right side of the page.  Click the one labelled `I need help`.  That will take you to a page that contains a section with the heading `License problem`.  Read what it says there & follow the instructions.  Don't stop at the first sentence.  Read the entire section.

**Comment by Sydeffect** [comment]
Bonjour 
J'ai le même problème j'aimerai savoir si vous avez une solution.
cordialement

**Reply by jcfrog** [reply]
bonjour, envoyez moi un email a licenses@downloadhelper.net avec l'adresse email utilisée pour l'achat.

**Comment by jcfrog** [comment]
> I am reaching out to DownloadHelper support because I am unable to recover my two keys.

Hi,
I sent you an email with both licenses.

**Comment by Sydeffect** [comment]
comme convenu voici un mail avec mon adresse utilisee pour ***@***.*** Cordialement Sydney BainsonSydney Bainson Le 3 août 2024 09:24, jcfrog ***@***.***> a écrit :

I am reaching out to DownloadHelper support because I am unable to recover my two keys.

Hi,
I sent you an email with both licenses.
